.coveragerc

```
@@ -2,7 +2,7 @@
 # https://coverage.readthedocs.io/en/latest/config.html
 
 [run]
-source = src/example
+source = src/mdyml
```

We have other code directories now:
```suggestion
source =
    src/md_from_template
    src/mdyml
    src/yml
    src/ymlmd
```